config/initializers/content_domain.rb

```diff
@@ -7,3 +7,11 @@
                      else
                        "https://www.gov.uk"
                      end
+
+ORGANISATION_URL = if domain.include?  "dev"
+                     "http://whitehall-admin.dev.gov.uk"
+                   elsif domain.include? "preview"
+                     "http://www.preview.alphagov.co.uk"
+                   else
+                     "https://www.gov.uk"
+                   end
```

Comment:
This logic seems more complex than it should need to be. `Plek.current.website_root` uses the `GOVUK_WEBSITE_ROOT` envvar, which is set from Puppet:

- development: http://www.dev.gov.uk
- preview: https://www.preview.alphagov.co.uk
- staging: https://www-origin.production.alphagov.co.uk
- production: https://www.gov.uk

It may be preferable to override that in dev, where it's useful to use the frontend app rather than assume we have the router running by using `http://www.dev.gov.uk`. The logic here uses `http` for preview, and points to production from staging, which could lead to confusion when deploying.

Although this is following the pattern of `MANUAL_CONTENT_URL`, could we simplify both here?